tests/wpunit/PostObjectSearchTest.php

```diff
@@ -0,0 +1,221 @@
+<?php
+
+class PostObjectSearchTest extends \Codeception\TestCase\WPTestCase {
+	public $current_time;
+	public $current_date;
+	public $current_date_gmt;
+	public $created_post_ids;
+	public $admin;
+	public $query;
+	public $app_context;
+	public $app_info;
+	public $subscriber;
+
+	public function setUp() {
+
+		parent::setUp();
+
+		global $wpdb;
+		$wpdb->delete( $wpdb->prefix . 'posts', array( 'post_type' => 'post' ) );
+
+		$this->current_time     = strtotime( '- 1 day' );
+		$this->current_date     = date( 'Y-m-d H:i:s', $this->current_time );
+		$this->current_date_gmt = gmdate( 'Y-m-d H:i:s', $this->current_time );
+		$this->admin            = $this->factory()->user->create( [
+			'role' => 'administrator',
+		] );
+		$this->subscriber = $this->factory()->user->create( [
+			'role' => 'subscriber'
+		]);
+
+		$this->created_post_ids = $this->create_posts();
+
+		$this->app_context = new \WPGraphQL\AppContext();
+
+		$this->app_info = new \GraphQL\Type\Definition\ResolveInfo( array() );
+
+		$this->query = '
+		query GET_POSTS($first: Int, $last: Int, $after: String, $before: String $where:RootQueryToPostConnectionWhereArgs) {
```

Comment:
@jasonbahl  seems like a comma is missing before the where arg

Reply:
good call. Commas in variable declarations are optional, so not a big deal, but probably should adjust for consistency 😄